pkg/actions/commands.go

```diff
@@ -16,23 +16,21 @@ import (
 	"net/http"
 	"os"
 
+	"github.com/eclipse/codewind-installer/pkg/appconstants"
 	"github.com/eclipse/codewind-installer/pkg/errors"
 	logr "github.com/sirupsen/logrus"
-
 	"github.com/urfave/cli"
 )
 
 var tempFilePath = "codewind-docker-compose.yaml"
 
-const versionNum = "x.x.dev"
-
 const healthEndpoint = "/api/v1/environment"
 
 //Commands for the controller
 func Commands() {
 	app := cli.NewApp()
 	app.Name = "cwctl"
-	app.Version = versionNum
+	app.Version = appconstants.VersionNum
```

Comment:
This is a good change. Out of scope for this PR but the rest of the const's should also be changed